api/core/types.proto

```diff
@@ -21,6 +21,9 @@ enum Kind {
     OCIRepository  = 7;
     Provider       = 8;
     Alert          = 9;
+    ImageRepository = 10;
+    ImageUpdateAutomation = 11;
```

Comment:
it looks a bit weird, all lines above this one has aligned `=` signs, but these two don't. Can we align the rest of it, or remove extra spaces everywhere so it's not aligned at all?